### PR TITLE
Created simple editor page

### PR DESCRIPTION
**Pull request by Taz-Z**

### CODE REVIEW

File: Project/src/main/java/app/FileUploadController.java [hunk 1]
```diff
@@ -45,7 +49,7 @@ public String uploadFile(@RequestParam("file") MultipartFile file) throws IOExce
         newFile.setFileType(file.getContentType());
         newFile.setData(file.getBytes());
         newFile.setUser(super.getUser());
-
+        newFile.setStatus(ArticleStatus.SUBMITTED.toString());
```
Comment (DanGravel) [inline Project/src/main/java/app/FileUploadController.java]: Is there a reason the article status has to be a string? Could we keep it as an enum

File: Project/src/main/resources/templates/editor.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            <tr th:each="file: ${files}">
+                <td th:text="${file.id}"></td>
+                <td th:text="${file.fileName}"></td>
+                <td th:text="${file.status}"></td>
+                <td th:text="${file.user.username}"></td>
+                <td th:text="${file.date}"></td>
+            </tr>
+        </tbody>
+    </table>
+
```
Comment (DanGravel) [inline Project/src/main/resources/templates/editor.html]: whitespace

File: Project/src/main/java/app/FileUploadController.java (hunk 1 above)
Comment (Taz-Z) [reply Project/src/main/java/app/FileUploadController.java]: It has to be displayed

File: Project/src/main/java/app/EditorController.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class EditorController extends app.Controller {
+    @Autowired
+    private FileRepository repository;
```
Comment (br0wncrust4cean) [inline Project/src/main/java/app/EditorController.java]: Just move this to the Controller super class. Can you abstract it from the UploadController as well?

**Review by br0wncrust4cean — CHANGES_REQUESTED** [review]

**Pull request merged**